### PR TITLE
Preparing for version 4.2.0

### PR DESCRIPTION
**Pull request by beylmk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 4.2.0
+
+- Add canMakePayments method (https://github.com/RevenueCat/react-native-purchases/pull/244)
+- Add PR template (https://github.com/RevenueCat/react-native-purchases/pull/263)
+
```
Comment (vegaro) [inline CHANGELOG.md:5]: The VERSIONS.md also needs to be updated

**Review by vegaro — CHANGES_REQUESTED** [review]

**Review by aboedo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (beylmk) [reply CHANGELOG.md:5]: 🤦 forgot to save changes in VS

File: VERSIONS.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 |---------|-------------|-----------------|----------------------|
-| 4.1.4   | 3.11.1      | 4.2.1           | 1.7.1                |
+| 4.2.0   | 3.11.1      | 4.2.1           | 1.7.1                |
```
Comment (vegaro) [inline VERSIONS.md:3]: It should be a new line with 4.2.0, without removing 4.1.4

**Review by vegaro — CHANGES_REQUESTED** [review]

File: VERSIONS.md (hunk 2 above)
Comment (beylmk) [reply VERSIONS.md:3]: 🤦

**Review by vegaro — APPROVED** [review]

**Pull request merged**